flowey/flowey_hvlite/src/pipelines/checkin_gates.rs

```diff
@@ -953,17 +953,31 @@ impl IntoPipeline for CheckinGatesCli {
             all_jobs.push(job);
         }
 
-        // Add a job that depends on all others as a workaround for https://github.com/orgs/community/discussions/12395.
-        // TODO: Add a way for this job to skip flowey setup and become a true no-op.
+        // Add a job that depends on all others as a workaround for
+        // https://github.com/orgs/community/discussions/12395.
+        //
+        // This workaround then itself requires _another_ workaround, requiring
+        // the use of `gh_dangerous_override_if`, and some additional custom job
+        // logic, to deal with https://github.com/actions/runner/issues/2566.
+        //
+        // TODO: Add a way for this job to skip flowey setup and become a true
+        // no-op.
         let all_good_job = pipeline
```

Comment:
I think we only need the "all good job" for PR builds, not CI right Otherwise this looks good to me!

Reply:
Fair point, that's an easy simplification. Done :)